components/eventing-controller/controllers/backend/reconciler.go

```diff
@@ -267,13 +236,17 @@ func (r *Reconciler) reconcileBEBBackend(ctx context.Context, bebSecret *v1.Secr
 	// Set environment with secrets for BEB subscription controller
 	err = setUpEnvironmentForBEBController(secretForPublisher)
 	if err != nil {
+		desiredStatus.SetSubscriptionControllerReadyCondition(false, eventingv1alpha1.ConditionReasonSubscriptionControllerNotReady, err.Error())
```

Comment:
(nit) `ConditionReasonSubscriptionControllerNotReady` does not seem appropriate here it's not ready because of an error. 

By "ConditionReasonSubscriptionControllerNotReady" message, I would understand that its in-progress w/o any errors.

Reply:
That's a fair point. I'll think of a new conditionReason then

Reply:
As discussed,  we will reuse the `ConditionReasonSubscriptionControllerStartFailed` Reason.